app/grants/templates/grants/card/index.html

```diff
@@ -16,13 +16,6 @@
 {% endcomment %}
 {% load i18n static humanize grants_extra %}
 
-<div class="grant-item flip-card">
-  <div class="flip-card__flipper">
-    <div class="flip-card__hold flip-card__front w-100">
-      {% include 'grants/card/front.html' %}
-    </div>
-    <div class="content-hover flip-card__hold flip-card__back">
-      {% include 'grants/card/back.html' %}
```

Comment:
Let's remove this file + css associated with flip-card__hold flip-card__back

Reply:
wait; why? we still have the grant card flips, no?